app/code/Magento/PageCache/Model/App/CacheIdentifierPlugin.php

```diff
@@ -15,6 +15,21 @@
  */
 class CacheIdentifierPlugin
 {
+    /**
+     * @var \Magento\PageCache\Model\Config
+     */
+    protected $config;
```

Comment:
please make properties with private visibility in plugin

Reply:
Resolved